### PR TITLE
Django Tutorial Part 8: User authentication and permissions

### PR DESCRIPTION
**Pull request by Fong113**

### CODE REVIEW

**Comment by Fong113** [comment]
ready

File: catalog/models.py [hunk 1]
```diff
@@ -53,11 +56,18 @@ class BookInstance(models.Model):
...
 
   def __str__(self):
     """String for representing the Model object."""
     return f'{self.id} ({self.book.title})'
 
+  @property
+  def is_overdue(self):
+    if self.due_back and date.today() > self.due_back:
+      return True
+    return False
```
Comment (huongnt-2545) [inline catalog/models.py:69]:
```suggestion
    return self.due_back and date.today() > self.due_back
```

**Comment by Fong113** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]

**Comment by huongnt-2545** [comment]
merged

**Pull request merged**